DEVELOPMENT.md

```diff
@@ -0,0 +1,78 @@
+# Developement
+
+## Feature Release Flow
+
+### Branching Strategy
+
+- The `master` branch is intended to only contain features for the immediately upcoming release, whether it is a Height Coordinated Upgrade or a Spork
+- During development, all features should live on a feature branch.
+- For small features, this will be simple working branch, i.e. `kan/1-fix-known-issue`
+- For larger features, this may be a shared feature branch with other team members.
+
+### Upgrade Path Eligibility
+
+- When a feature branch is ready to be merged, it must be determined if the desired upgrade path onto Mainnet. The options are:
+    - Height Coordinated Upgrade (HCU)
+        - No protocol level breaking changes
+        - No state migrations
+        - Execution state/path changes allowed
```

Comment:
What does this mean in particular. Changes to how execution state is stored need a state migration / spork. 
Maybe clarify to avoid potential misunderstanding and problems due to it.